.github/workflows/pull-compliance_docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Compliance testing for documentation
+
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+      - "*.md"
+
+jobs:
+  compliance-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup node
+        uses: actions/setup-node@v2
```

Comment:
`setup-node` is already at `v3`, but I guess I will fix that with the upcoming node 20 upgrade.